ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -120,6 +120,15 @@ svi_profiles:
     # This setting can be overridden per VRF.
     enable_mlag_ibgp_peering_vrfs: < true | false >
 
+    # EVPN L2 Multicast per SVI | Optional
+    # Configures multicast vxlan vlan multicast group ipv4 address using the following formula: < evpn_l2_multicast.vxlan_multicast_group_ipv4_pool > + < vlan_id - 1 >.
+    # Recommendation to assign a /20 block within 232.0.0.0/16 Source Specific Multicast range.
+    # Enables `redistribute igmp` on the router bgp MAC VRF.
+    evpn_l2_multicast:
+      enabled: < true | false >
+      vxlan_multicast_group_ipv4_pool: < IPv4_address/Mask >
```

Comment:
```suggestion
      underlay_l2_multicast_group_ipv4_pool: < IPv4_address/Mask >
```

Reply:
idea is that for L3 Multicast there could be knob:
`underlay_vrf_multicast_group_ipv4_pool: < IPv4_address/Mask >`